libs/application/templates/financial-aid/src/lib/messages/error.ts

```diff
@@ -7,6 +7,16 @@ export const error = {
       defaultMessage: 'Samþykkja þarf gagnaöflun til að halda áfram',
       description: 'Error message when data gathering is not approved',
     },
+    radioErrorMessage: {
+      id: 'fa.application:section.radioErrorMessage',
+      defaultMessage: 'Þú þarft að velja einn valmöguleika',
+      description: 'When radio is not selected',
```

Comment:
Should we have this more descriptive? Since content managment is hard. For example should this be `Error message when no option is selected` 